modules/tests/test/create_vm_common_test.go

```diff
@@ -155,7 +153,7 @@ var _ = Describe("Create VM", func() {
 								CreateVMResults.Namespace: expectedVM.Namespace,
 							})
 
-						vm, err := vm.WaitForVM(f.KubevirtClient, f.CdiClient, expectedVM.Namespace, expectedVM.Name,
+						vm, err := vm.WaitForVM(f.K8sClient, f.KubevirtClient, f.CdiClient, expectedVM.Namespace, expectedVM.Name,
```

Comment:
Can't the `kubecli.KubevirtClient` be used for everything? It implements `kubernetes.Clientset` and provides the CDI client with method `CdiClient()`.